Project.toml

```
@@ -21,24 +21,24 @@ SparseArrays = "2f01184e-e22b-5df5-ae63-d93ebab69eaf"
 [compat]
 julia = "1.0, 1.1, 1.2, 1.3"
 CDDLib = "0.5.3"
-Distributions = "0.19.2"
+Distributions = "^0.19, ^0.20, ^0.21"
 Documenter = "0.23.0"
-Expokit = "≥ 0.1.0"
-GLPK = "≥ 0.10.0"
-GLPKMathProgInterface = "≥ 0.4.0"
-GR = "≥ 0.39.1"
-IntervalArithmetic = "≥ 0.14.0"
-IntervalMatrices = "≥ 0.2.1"
-JuMP = "0.20.0"
-Makie = "≥ 0.9.4"
-MathProgBase = "≥ 0.7.0"
-Optim = "≥ 0.14.1"
-Plots = "≥ 0.25.1"
-Polyhedra = "0.5.4"
-RecipesBase = "≥ 0.3.0"
-Reexport = "≥ 0.2.0"
-Requires = "≥ 0.4.0"
-TaylorModels = "≥ 0.1.1"
+Expokit = "^0.2"
+GLPK = "^0.10, ^0.11"
+GLPKMathProgInterface = "^0.4"
+GR = "^0.41"
```

```suggestion
GR = "^0"
```
Your version is already outdated. Since this package is completely optional and I do not expect any breaking changes before v1 because we only use it through `Plots`, I say we do not need to add a bound for the minor version.